buildpack.md

```diff
@@ -162,12 +166,38 @@ Executable: `/bin/build <layers[EIC]> <platform[AR]> <plan[ER]>`, Working Dir: `
 
 Using the [Layer Content Metadata](#layer-content-metadata-toml) provided by a buildpack in a `<layers>/<layer>.toml` file, the lifecycle MUST determine:
 
-- Whether the layer directory in `<layers>/<layer>/` should be available to the app (via the `launch` boolean).
-- Whether the layer directory in `<layers>/<layer>/` should be available to subsequent buildpacks (via the `build` boolean).
-- Whether and how the layer directory in `<layers>/<layer>/` should be persisted to subsequent builds of the same OCI image (via the `cache` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be available to the app (via the `launch` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be available to subsequent buildpacks (via the `build` boolean).
+- Whether the layer directory `<layers>/<layer>/` should be persisted and made available to subsequent builds of the same OCI image (via the `cache` boolean).
 
 All combinations of `launch`, `build`, and `cache` booleans are valid. When a layer declares more than one type (e.g. `launch = true` and `cache = true`), the requirements of each type apply.
 
+`build`   | `cache`  | `launch` | Metadata Restored        | Layer Restored      
+----------|----------|----------|--------------------------|---------------------
+T         | T        | T        | Yes - from the app image | Yes - from the cache
+T         | T        | F        | Yes - from the cache     | Yes - from the cache
+T         | F        | T        | No                       | No
+T         | F        | F        | No                       | No
+F         | T        | T        | Yes - from the app image | Yes - from the cache
+F         | T        | F        | Yes - from the cache     | Yes - from the cache
+F         | F        | T        | Yes - from the app image | No
+F         | F        | F        | No                       | No
+
+Notes:
+* The metadata and layer are restored only if they match each other (the cache can be changed/deleted).
+* The layer is restored only from the cache due to performance reasons.
+
+Examples:
+* `build == T, cache == T, launch == F`: Java buildpack.  
+If the version of the jdk in the cache is the one that is needed, a downstream buildpack will use it.  
```

Comment:
```suggestion
Example: a Java buildpack might read the restored layer metadata to determine if the version of the JDK used in the previous build of the OCI image is the one that is needed. If so, it might choose to re-use the layer from the cache to avoid re-downloading.
```

Reply:
Please see https://github.com/buildpacks/spec/pull/209/commits/35eda43c391bcbbdd15fc7858ed9ec7f9e865f3d.